### PR TITLE
D3ASIM-3615: Adding PV configuration validator.

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

File: d3a_interface/validators/pv_validator.py [hunk 1]
```diff
@@ -114,3 +115,22 @@ def validate_rate(cls, **kwargs):
...
+                "to work with PV orientations."]})
+
+        # validate azimuth isn't out of bound
+        if kwargs.get("azimuth") is not None and not 0 <= kwargs["azimuth"] <= 360:
+            raise D3ADeviceException({"misconfiguration": ["Azimuth could only be in between "
+                                                           "0 and 360 degrees."]})
+
+        # validate tilt isn't out of bound
+        if kwargs.get("tilt") is not None and not 0 <= kwargs["tilt"] <= 90:
+            raise D3ADeviceException({"misconfiguration": ["tilt could only be in between "
```
Comment (hannesdiedrich) [inline d3a_interface/validators/pv_validator.py:135]: Why not sticking to how we are doing it already, by using `Rangelimit` in the `constants_limits` module, e.g. https://github.com/gridsingularity/d3a-interface/blob/f7ac331728a0b650623796c72f40efae69162c27/d3a_interface/constants_limits.py#L127 ?

Comment (faizan2590) [reply d3a_interface/validators/pv_validator.py:135]: Thanks, forgot about that, will do it now

**Review by Hassan754 — APPROVED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**